README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Go Report Card](https://goreportcard.com/badge/kubernetes-sigs/cluster-api-provider-azure)](https://goreportcard.com/report/kubernetes-sigs/cluster-api-provider-azure)
 
-<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png"  width="100">
+<img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png"  width="100"><a href="https://azure.microsoft.com/en-us/overview/open-source/"><img src="https://azure.microsoft.com/svghandler/information-protection/?width=200&height=110" alt="Powered by Azure"></a>
```

Comment:
there is actually a new Azure logo since May 7th https://azure.microsoft.com/en-us/blog/a-fluent-new-look-for-the-azure-icon/

Reply:
Oh okay! Do u know where I can found a png or svg of the new logo :D?

Reply:
Updated now with new azure logo 😄  thanks!

Reply:
Awesome. What do you think about adding a bit of space between the two logos so it's a little more spread out?

Reply:
also, where did you end up getting the new logo from? Poked around a bit and couldn't find the svg file anywhere

Reply:
I also couldn't find the svg :c so I took the image from the Github Azure profile picture https://github.com/Azure

Reply:
Space between added 👍🏻 !